packages/modules/cra-template-itwin-viewer/README.md

```diff
@@ -0,0 +1,21 @@
+# cra-template-itwin-viewer
+
+This is a template for applications that are based on the [iTwin Viewer](https://github.com/imodeljs/itwin-viewer/tree/master/packages/modules/itwin-viewer-react) for [Create React App](https://github.com/facebook/create-react-app).
+
+To use this template, add `--template @bentley/itwin-viewer` when creating a new app. You should also use the @bentley/react-scripts scripts version to compile your application (i.e. `npx create-react-app my-app-name --template @bentley/itwin-viewer --scripts-version @bentley/react-scripts`).
+
+For example:
+
+```sh
+npx create-react-app my-app --template itwin-viewer
```

Comment:
Is it `itwin-viewer` or `@bentley/itwin-viewer` currently?  I think we could publish the package either way, I don't think anyone already has `itwin-viewer` outside of our scope

Reply:
it should be scoped. CRA docs say it will work but may need to update once published if not. I'll update for now.

Reply:
fixed